locale/fr.po

```diff
@@ -0,0 +1,70 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-06-26 12:18+0200\n"
+"PO-Revision-Date: 2017-06-26 12:22+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: fr_FR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.0.2\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: init.lua:19
+msgid "Steel blacksmithing hammer"
+msgstr "Marteau de forgeron en acier"
+
+#: init.lua:20
+msgid "A tool for repairing other tools at a blacksmith's anvil."
+msgstr "Un outil pour réparer les autres outils avec une enclume de forgeron."
+
+#: init.lua:21
+msgid ""
+"Use this hammer to strike blows upon an anvil bearing a damaged tool and you "
+"can repair it. It can also be used for smashing stone, but it is not well "
+"suited to this task."
+msgstr ""
+"Utilisez ce marteau pour frapper une enclume contenant un outil endommage, "
+"ainsi vous pourrez le réparer. Il peut etre aussi utiliser pour casser de la "
+"pierre, mais il n'est pas adapté à cette tâche."
```

Comment:
utilisé

Reply:
Which "use" is mistranslated here, Utilisez or utiliser?

Reply:
The second one ("utiliser"). The reason for that is that you are using the past infinite form of the verb ("être utilisé"), which needs to have the auxiliary ("être") in its infinitive form, and the verb itself in its past participle form (i.e. like "It can be used", where "be" is an infinitive, and "used" is a past participle).
I just saw you also need to change "etre" -> "être" as well.